tests/test_xlsx_validation.py

```diff
@@ -10,16 +10,28 @@ class TestEvaXlsValidator(TestCase):
     def setUp(self) -> None:
         brokering_folder = os.path.join(ROOT_DIR, 'tests', 'resources', 'brokering')
         metadata_file = os.path.join(brokering_folder, 'metadata_sheet2.xlsx')
+        metadata_file_fail = os.path.join(brokering_folder, 'metadata_sheet_fail.xlsx')
         self.validator = EvaXlsxValidator(metadata_file)
+        self.validator_fail = EvaXlsxValidator(metadata_file_fail)
 
     def test_cerberus_validation(self):
         self.validator.cerberus_validation()
-        print(self.validator.error_list)
+        self.assertEqual(self.validator.error_list, [])
+
+        self.validator_fail.cerberus_validation()
+        expected_errors = ['In Sheet Analysis, Row 4, field Analysis Alias: null value not allowed']
+        self.assertEqual(self.validator_fail.error_list, expected_errors)
```

Comment:
I think in general it's a good idea to test one scenario per test, i.e. split this into `test_cerberus_validation` and `test_cerberus_validation_failure` (and similar for the following test).  This just makes it a bit easier to trace where the issue is if a test breaks.

Reply:
Changed these tests. But it's probably something I have done in other places. We can split them as we come across them.